src/main/java/uk/gov/hmcts/darts/hearings/controller/HearingsController.java

```diff
@@ -52,8 +52,8 @@ public ResponseEntity<List<EventResponse>> getEvents(Integer hearingId) {
     @Override
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(contextId = HEARING_ID,
-        securityRoles = {JUDGE, ADMIN, REQUESTER, APPROVER, TRANSCRIBER},
-        globalAccessSecurityRoles = {JUDGE, ADMIN})
+        securityRoles = {JUDGE, ADMIN, REQUESTER, APPROVER, TRANSCRIBER, RCJ_APPEALS},
+        globalAccessSecurityRoles = {JUDGE, ADMIN, RCJ_APPEALS})
```

Comment:
Is it correct to add `RCJ_APPEALS` here?

Reply:
hey Dave, the spreadsheet has two rows 'View Hearing Details - Transcript Tab' and underneath that 'View Available Transcript' with 'RCJ_APPEALS' having permission (See image I've attached).
This is the corresponding endpoint used by the UI so I added 'RCJ_APPEALS'. 
Please let me know if this is wrong

Reply:
Looks fine 👍  I was only asking because it wasn't the `TRANSLATION_QA` role.

Reply:
oh I see. 
I double checked also the RCJ_APPEALS role since it is in a column adjacent to TRANSLATION_QA :)